package.json

```diff
@@ -65,13 +65,14 @@
     "body-parser": "^1.15.2",
     "compression": "^1.6.2",
     "cookie-parser": "^1.4.3",
-    "copyfiles": "^1.0.0",
+    "copyfiles": "^1.2.0",
     "cross-env": "^3.1.4",
     "csurf": "^1.9.0",
     "express": "^4.14.0",
     "express-force-ssl": "^0.3.2",
     "file-loader": "^0.10.0",
-    "history": "3.0.0",
+    "history": "^4.6.1",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
Could you please make that change (replacing `axios` by `isomorphic-fetch`) on another PR? It's actually a bug we missed.